server/static/index.html

```diff
@@ -15,7 +15,7 @@
                 <div class="hole hole-bottom-right"></div>
                 <div class="logo" hx-get="/htmx/meta" hx-swap="innerHTML" hx-trigger="load"></div>
                 <div id="status-holder">
-                    <div id="status-lcd" class="led led-off" hx-get="/htmx/status" hx-trigger="load" hx-swap="outerHTML">88888888888888</div>
+                    <div id="status-lcd" class="led led-off" hx-get="/htmx/status" hx-trigger="load" hx-swap="outerHTML"><marquee behavior="alternate" scrollamount="3">88888888888888</marquee></div>
```

Comment:
Should not use marquee tag as that is deprecated and there are better ways to do this in css